obj.go

```diff
@@ -297,6 +331,23 @@ func (t ObjectType) NewObject() (*Object, error) {
 	return &Object{ObjectType: t, dpiObject: obj}, nil
 }
 
+// Close releases a reference to the object type.
+func (t ObjectType) Close() error {
+
+	for _, attr := range t.Attributes {
+		err := attr.Close()
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
t.Attributes = nil
d := t.dpiObjectType
t.dpiObjectType = nil
if d == nil {
return nil
}
// and use d instead of t.dpiObjectType

Reply:
Done